resources/views/includes/structure/footer.blade.php

```diff
@@ -172,7 +181,7 @@
                     </a>
                     <a aria-label="Fitzwilliam Museum sketchfab account"
                         href="https://www.sketchfab.com/fitzwilliammuseum/">
-                        <img src="{{ asset( "/images/logos/sketchfab-logo.svg") }}" width="48" height="48"
+                        <img src="{{ asset( " /images/logos/sketchfab-logo.svg") }}" width="48" height="48"
```

Comment:
Is the space here after the quotation marks necessary? If not, probably best to remove it for consistency with the rest of the site